obcsim/obcsim_configuration.hpp

```diff
@@ -49,6 +49,12 @@
 #ifndef MSP_OP_SEND_CUBES_GATEWARE_CONF
 #define MSP_OP_SEND_CUBES_GATEWARE_CONF 0x77
 #endif
+#ifndef MSP_OP_SEND_NVM_CITI_CONF
+#define MSP_OP_SEND_NVM_CITI_CONF 0x79
+#endif
+#ifndef MSP_OP_APPLY_CUBES_CITI_CONF
+#define MSP_OP_APPLY_CUBES_CITI_CONF 0x7A
```

Comment:
Should be `MSP_OP_SELECT_CUBES_CITI_CONF`, to be in concordance to the definition in `cubes-fw`. (Sorry, I guess I missed this last time.)